dashboard/modules/serve/serve_rest_api_impl.py

```diff
@@ -213,6 +215,41 @@ def validate_http_options(self, client, http_options):
                     f"updated: {divergent_http_options}."
                 )
 
+        def log_config_change_default_warning(self, config):
+            from ray.serve.config import AutoscalingConfig
+
+            for deployment in [
+                d for app in config.applications for d in app.deployments
+            ]:
+                if "max_ongoing_requests" not in deployment.dict(exclude_unset=True):
+                    logger.warning(
+                        "The default value for `max_ongoing_requests` has changed "
+                        "from 100 to 5."
+                    )
```

Comment:
It's not obvious to a new user that this change took place because of the new Ray version. The log statement sounds like something dynamically changed the default during runtime.

```suggestion
                    logger.warning(
                        "The default value for `max_ongoing_requests` has changed "
                        "from 100 to 5 in Ray 2.xx"
                    )
```

Reply:
Good point, set it to ray 2.31